### PR TITLE
added function to get create food and send it to fe

### LINKED ISSUE #4: Make the backend websocket server send food location
Make the backend websocket server send list of food location and direction on initial connection with the client. The list should be like 
[[x of food1, y of food1, direction of food1],[x of food2, y of food2, ...........................]

the data should be like 
{ food : listoffoods}

### PR DESCRIPTION
**Pull request by kcaashish**
Just added create food function

### CODE REVIEW

**Comment by anirbanmajumder0** [comment]
Send the food list to the client after initial connection with the client

**Review by Anirban-Majumder — CHANGES_REQUESTED** [review]
Send the food list to the client after initial connection with the client. And stop sending the dummy data.

File: backend/main.py [hunk 1]
```diff
@@ -11,20 +16,38 @@ async def websocket_endpoint(websocket: WebSocket) -> None:
...
+    """
+    food_list = []
+    for _ in range(3):
+        ran = random.SystemRandom()
+        r = math.floor(ran.random() * 40)
+        food_x = r * BLOCK_SIZE
+        r = math.floor(ran.random() * 40)
+        food_y = r * BLOCK_SIZE
+        food_direction = math.floor(ran.random() * 3)
+        food_list.append([food_x, food_y, food_direction])
```
Comment (Anirban-Majumder) [inline backend/main.py]: Make this into a separate function cause the client will call the server for new food when old food is eaten.

Comment (kcaashish) [reply backend/main.py]: Done

**Review by Anirban-Majumder — APPROVED** [review]

**Pull request merged**